docs/starters.yml

```diff
@@ -1609,3 +1609,15 @@
     - Progressive Web app
     - Offline Support
     - RSS Feed
+- url: https://ueno.co/
+  repo: https://github.com/ueno-llc/ueno-gatsby-starter
+  description: Ueno's starter template for Gatsby
+  tags:
+    - Ueno
```

Comment:
Sorry, but we don't accept tags referencing single companies/agencies etc. They should stay meaningful and enable the user to have a good overview. So I kindly ask you to remove that 👍 

```suggestion
```

Reply:
Fixed 👍